android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/Keyboard.java

```diff
@@ -156,22 +158,39 @@ public JSONObject toJSON() {
     return o;
   }
 
-  // Default sil_euro_latin keyboard
+  /**
+   * Get the fallback keyboard. If never specified, use sil_euro_latin
+   * @param context Context
```

Comment:
Is it legal to call this with a `null` context? If not, then we should assert that `context` is not null on entry and then we can be sure that it always returns a fallback keyboard.